sql/core/src/test/scala/org/apache/spark/sql/DataFrameTimeWindowingSuite.scala

```diff
@@ -521,7 +521,7 @@ class DataFrameTimeWindowingSuite extends QueryTest with SharedSparkSession {
     Seq(df1, df2, df3, df4).foreach { df =>
       val filter = df.queryExecution.optimizedPlan.find(_.isInstanceOf[Filter])
       assert(filter.isDefined)
-      val exist = filter.get.constraints.filter(e =>
+      val exist = filter.get.constraints.filter( e =>
```

Comment:
nit: Previous was correct based on style guide. please refer below guidance:
https://github.com/databricks/scala-style-guide#anonymous-methods

When you use parentheses for anonymous method, no space. When you use curly braces for anonymous method, space.

For multi-lines we prefer using curly braces, like

```
val exist = filter.get.constraints.filter { e =>
  e.toString.contains(">=") || e.toString.contains("<")
}
```

Reply:
Has been changed back. Thanks!